### ISSUE TITLE
Resource Lists

### ISSUE THREAD

**Issue by caro27** [cu-mkp/research-teaching-companion#147]
Can be hard to find specific presentations or activity sheets under the right menu or sort through all the (incredible) context we provide. Perhaps a list of "all presentations," "all instructor documents," "all student handouts" would be useful. We wouldn't need to provide any additional information. 
I think it might streamline process for users. For example, today I was looking for all presentations in the RTC and having to click and carefully read each page takes time and you can miss things amidst all the other things on the page.

**Comment by ps2270** [comment]
I agree! But it would have to be carefully titled and constructed so it doesn't become another confusing resource. Why don't you start google doc that mocks up that possible md page for "All Resources for Hands-On Lesson Plans" that could be linked at the start of the Hands-On Lesson Plans md index page.
